src/Library/demos/Drawing Area/main.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Drawing Area",
+  "category": "user_interface",
+  "description": "Connection between control and drawing.",
+  "panels": [
+    "ui",
```

Comment:
let's put it here for now, I think we'll have a Graphics or Multimedia section later